File: staylio_backend/src/main/java/com/example/staylio_backend/dto/response/HotelBranchResponse.java
package com.example.staylio_backend.dto.response;

import com.example.staylio_backend.model.enums.BranchStatus;
import lombok.Builder;
import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
@Builder
public class HotelBranchResponse {
    private Long id;
    private Long provinceId;
    private Long wardId;
    private String hotelBranchName;
    private String address;
    private String imageUrl;
    private String wardName;
    private String provinceName;
    private Long hotelId;
    private String hotelName;
    private String description;
    private String phone;
    private Integer capacity;
    private BranchStatus status;
    private Long countReview;
    private Double averageRating;
    private Double latitude;
    private Double longitude;
    private String managerName;
}

File: staylio_frontend/common/interfaces/response/HotelBranchResponse.ts
import { BranchStatus } from "@common/enums/BranchStatus";

export interface HotelBranchResponse {
    id: number;
    provinceId: number;
    wardId: number;
    hotelBranchName: string;
    hotelId: number;
    hotelName: string;
    imageUrl: string | null;
    address: string;
    provinceName: string;
    wardName: string;
    capacity: number;
    phone: string;
    description: string;
    status: BranchStatus;
    countReview: number;
    averageRating: number;
    latitude: number;
    longitude: number;
}